src/drc/api/tests/test_audittrails.py

```diff
@@ -0,0 +1,199 @@
+import uuid
+from base64 import b64encode
+from unittest.mock import patch
+from freezegun import freeze_time
+from datetime import datetime
+
+from django.test import override_settings
+from django.urls import reverse, reverse_lazy
```

Comment:
you can use `from vng_api_common.tests import reverse, reverse_lazy`

These are overloaded, and you can do for example: `reverse_lazy(EnkelVoudigInformatieObject)` which will give you the list url. If you pass an instance, it'll give you the detail URL. It already reads out the settings for version information.